include/dd/Package.hpp

```diff
@@ -595,6 +638,39 @@ namespace dd {
             return makeDDNode<vNode>(level, {zeroSuccessor, oneSuccessor}, true);
         }
 
+        /**
+        Converts a vectorized matrix into a decision diagram starting from the top qubit level.
+                @param begin an iterator to the beginning of the range of complex numbers representing the matrix.
+                @param end an iterator to the end of the range of complex numbers representing the matrix.
+                @param level the qubit level at which to start building the DD.
+                @return a DD representing the matrix.
+        **/
```

Comment:
this docstring needs to be updated to reflect the new implementation

Reply:
Done